pandas/core/reshape/melt.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=E1101,E1103
+# pylint: disable=W0703,W0622,W0613,W0201
+import numpy as np
+
+from pandas.core.dtypes.common import is_list_like
+from pandas import compat
+
+from pandas.core.frame import DataFrame
```

Comment:
import these from pandas

Reply:
reshape.py uses the full path. Also, I'm getting an import error when I do `from pandas import DataFrame`